flink-core/src/main/java/org/apache/flink/configuration/MetricOptions.java

```diff
@@ -272,6 +272,17 @@ public static Configuration forReporter(Configuration configuration, String repo
                     .withDescription(
                             "Defines the scope format string that is applied to all metrics scoped to a job on a JobManager. Only effective when a identifier-based reporter is configured");
 
+    /**
+     * The scope format string that is applied to all metrics scoped to the components running on a
+     * JobManager of an operator.
+     */
+    public static final ConfigOption<String> SCOPE_NAMING_JM_OPERATOR =
+            key("metrics.scope.jm-operator")
+                    .stringType()
+                    .defaultValue("<host>.jobmanager.<job_name>.<operator_name>")
```

Comment:
The documentation must be regenerated.

Additionally the metrics docs page must be updated.

Reply:
I will add the docs about it.